jmix-flowui/flowui-themes/src/main/resources/META-INF/resources/themes/jmix-lumo/component/vaadin-checkbox-group.css

```diff
@@ -0,0 +1,4 @@
+vaadin-checkbox-group[readonly][has-error-message]::part(error-message),
```

Comment:
Even though, Vaadin for some reason makes checkboxGroup `disabled` in readonly mode, it'd better to handle error indicator color. It may appear that Vaadin adjust readonly style to match `RadioButtonGroup`.